package.json

```diff
@@ -16,29 +16,34 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-3": "^6.24.1",
+    "bluebird": "^3.5.1",
     "body-parser": "^1.18.2",
+    "chai": "^4.1.2",
```

Comment:
Heroku install only dependencied in default. In order to run test, moved these modules from devDependencies.